develop/tutorials/articles/140-application-security/01-application-permissions/03-registering-permissions.markdown

```diff
@@ -122,8 +122,15 @@ class.
 
         }
 
-The `activate` 
+The `activate` method is a *bundle activator*. Normally in Declarative Services
```

Comment:
It's a component activator, bundle activators are something different.

Reply:
Missed this one; fixed in the next commit.